interpreter/syntax/types.ml

```diff
@@ -81,11 +81,20 @@ let as_syn_var = function
   | SynVar x -> x
   | SemVar _ -> assert false
 
+let as_sem_var = function
+  | SynVar _ -> assert false
+  | SemVar x -> x
+
 let as_func_def_type (dt : def_type) : func_type =
   match dt with
   | FuncDefType ft -> ft
   | _ -> assert false
 
+let as_cont_def_type (dt : def_type) : cont_type =
+  match dt with
+  | ContDefType x -> x
```

Comment:
nitpick: change `x` to `ft`

Reply:
Actually, should be `ct`, since it's a cont_type. Fixed.

Reply:
Of course... I wasn't sure if a `cont_type` was just a `func_type`, but obviously not.